<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Tester-PC" failures="9" tests="41" name="Master_Employee" time="99.094" errors="0" timestamp="21 Aug 2018 05:58:24 GMT">
  <testcase classname="com.mytest.Master_Employee" name="Employeename_field_is_Blank" time="51.034"/>
  <testcase classname="com.mytest.Master_Employee" name="Employeename_contain_special_characters" time="0.299">
    <failure type="java.lang.AssertionError" message="Employee Name Contains Special Characters expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Employee Name Contains Special Characters expected [true] but found [false]
at com.mytest.Master_Employee.Employeename_contain_special_characters(Master_Employee.java:41)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Employeename_contain_special_characters -->
  <testcase classname="com.mytest.Master_Employee" name="Employeename_contain_numbers" time="0.192"/>
  <testcase classname="com.mytest.Master_Employee" name="Employeename_contain_alphabets" time="0.203"/>
  <testcase classname="com.mytest.Master_Employee" name="Mobile_No_contain_special_characters" time="0.327">
    <failure type="java.lang.AssertionError" message=" Mobile No Contains Spaecial Characters expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  Mobile No Contains Spaecial Characters expected [true] but found [false]
at com.mytest.Master_Employee.Mobile_No_contain_special_characters(Master_Employee.java:68)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Mobile_No_contain_special_characters -->
  <testcase classname="com.mytest.Master_Employee" name="Mobile_No_contain_numbers" time="0.23"/>
  <testcase classname="com.mytest.Master_Employee" name="Mobile_No_contain_whitespaces" time="0.226"/>
  <testcase classname="com.mytest.Master_Employee" name="Mobile_No_contain_alphabets" time="0.16"/>
  <testcase classname="com.mytest.Master_Employee" name="Mobile_No_contain_whitespaces_at_begining" time="0.219"/>
  <testcase classname="com.mytest.Master_Employee" name="Mobile_No_field_is_Blank" time="0.081"/>
  <testcase classname="com.mytest.Master_Employee" name="Mobile_No_contain_lessthan_10_numbers" time="0.172">
    <failure type="java.lang.AssertionError" message=" Mobile No Contains Less than 10 numbers expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  Mobile No Contains Less than 10 numbers expected [true] but found [false]
at com.mytest.Master_Employee.Mobile_No_contain_lessthan_10_numbers(Master_Employee.java:130)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Mobile_No_contain_lessthan_10_numbers -->
  <testcase classname="com.mytest.Master_Employee" name="Mobile_No_contain_greaterthan_10_numbers" time="0.229"/>
  <testcase classname="com.mytest.Master_Employee" name="Designation_contain_special_characters" time="0.278">
    <failure type="java.lang.AssertionError" message="Employee Name Contains Special Characters expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Employee Name Contains Special Characters expected [true] but found [false]
at com.mytest.Master_Employee.Designation_contain_special_characters(Master_Employee.java:148)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Designation_contain_special_characters -->
  <testcase classname="com.mytest.Master_Employee" name="Designation_contain_numbers" time="0.194">
    <failure type="java.lang.AssertionError" message="Employee Name Contains Numbers expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Employee Name Contains Numbers expected [true] but found [false]
at com.mytest.Master_Employee.Designation_contain_numbers(Master_Employee.java:157)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Designation_contain_numbers -->
  <testcase classname="com.mytest.Master_Employee" name="Designation_contain_alphabets" time="0.176"/>
  <testcase classname="com.mytest.Master_Employee" name="Email_contains_whitespaces" time="3.778"/>
  <testcase classname="com.mytest.Master_Employee" name="Email_valid" time="0.264"/>
  <testcase classname="com.mytest.Master_Employee" name="Email_contains_whitespaces_at_begining" time="1.481">
    <failure type="java.lang.AssertionError" message=" Email contain Whitespaces at Begining expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  Email contain Whitespaces at Begining expected [true] but found [false]
at com.mytest.Master_Employee.Email_contains_whitespaces_at_begining(Master_Employee.java:200)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Email_contains_whitespaces_at_begining -->
  <testcase classname="com.mytest.Master_Employee" name="Incentive_contain_special_characters" time="0.221"/>
  <testcase classname="com.mytest.Master_Employee" name="Incentive_contain_whitespaces" time="0.234"/>
  <testcase classname="com.mytest.Master_Employee" name="Incentive_contain_alphabets" time="0.283"/>
  <testcase classname="com.mytest.Master_Employee" name="Incentive_contain_numbers" time="0.189"/>
  <testcase classname="com.mytest.Master_Employee" name="Incentive_contain_negative_numbers" time="0.186"/>
  <testcase classname="com.mytest.Master_Employee" name="Incentive_contain_whitespaces_at_begining" time="0.217"/>
  <testcase classname="com.mytest.Master_Employee" name="Blank_Gender_field" time="0.209">
    <failure type="java.lang.AssertionError" message="Gender field is Blank expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Gender field is Blank expected [true] but found [false]
at com.mytest.Master_Employee.Blank_Gender_field(Master_Employee.java:264)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Blank_Gender_field -->
  <testcase classname="com.mytest.Master_Employee" name="Gender_functionality" time="0.296"/>
  <testcase classname="com.mytest.Master_Employee" name="Valid_Address" time="0.528"/>
  <testcase classname="com.mytest.Master_Employee" name="PersonalInfo_Next_Button_Functionality" time="1.173"/>
  <testcase classname="com.mytest.Master_Employee" name="Blank_UserName" time="0.165"/>
  <testcase classname="com.mytest.Master_Employee" name="Existing_UserName" time="1.414"/>
  <testcase classname="com.mytest.Master_Employee" name="UserName" time="0.258"/>
  <testcase classname="com.mytest.Master_Employee" name="Blank_Employee_Role" time="0.051">
    <failure type="java.lang.AssertionError" message="Employee Role is Blank expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Employee Role is Blank expected [true] but found [false]
at com.mytest.Master_Employee.Blank_Employee_Role(Master_Employee.java:332)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Blank_Employee_Role -->
  <testcase classname="com.mytest.Master_Employee" name="Employee_Role" time="1.694"/>
  <testcase classname="com.mytest.Master_Employee" name="Blank_Password" time="0.068"/>
  <testcase classname="com.mytest.Master_Employee" name="Password" time="0.141"/>
  <testcase classname="com.mytest.Master_Employee" name="Blank_Confirm_Password" time="0.088"/>
  <testcase classname="com.mytest.Master_Employee" name="Different_Confirm_Password" time="0.241"/>
  <testcase classname="com.mytest.Master_Employee" name="Confirm_Password" time="0.167"/>
  <testcase classname="com.mytest.Master_Employee" name="EmployeeRole_Next_Button_Functionality" time="1.166"/>
  <testcase classname="com.mytest.Master_Employee" name="Finish_Button_Functionality" time="8.012"/>
  <testcase classname="com.mytest.Master_Employee" name="Update_Button_Functionality" time="22.433">
    <failure type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&amp;quot;method&amp;quot;:&amp;quot;xpath&amp;quot;,&amp;quot;selector&amp;quot;:&amp;quot;//*[@id=&amp;quot;btn-save&amp;quot;]&amp;quot;}
  (Session info: chrome=68.0.3440.106)
  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &amp;apos;3.13.0&amp;apos;, revision: &amp;apos;2f0d292&amp;apos;, time: &amp;apos;2018-06-25T15:32:14.902Z&amp;apos;
System info: host: &amp;apos;TESTER-PC&amp;apos;, ip: &amp;apos;192.168.0.100&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;10.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.40.565498 (ea082db3280dd6..., userDataDir: C:\Users\Tester\AppData\Loc...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 68.0.3440.106, webStorageEnabled: true}
Session ID: f128d33a98524b62303bd33105cc7d38
*** Element info: {Using=xpath, value=//*[@id=&amp;quot;btn-save&amp;quot;]}">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="btn-save"]"}
  (Session info: chrome=68.0.3440.106)
  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:32:14.902Z'
System info: host: 'TESTER-PC', ip: '192.168.0.100', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '10.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.40.565498 (ea082db3280dd6..., userDataDir: C:\Users\Tester\AppData\Loc...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 68.0.3440.106, webStorageEnabled: true}
Session ID: f128d33a98524b62303bd33105cc7d38
*** Element info: {Using=xpath, value=//*[@id="btn-save"]}
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:322)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:424)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:314)
at com.mytest.Master_Employee.Update_Button_Functionality(Master_Employee.java:455)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Update_Button_Functionality -->
  <testcase classname="com.mytest.Login" name="login" time="0.0">
    <ignored/>
  </testcase> <!-- login -->
  <testcase classname="com.mytest.Master_Employee" name="Mobile_No_contain_negative_numbers" time="0.0">
    <ignored/>
  </testcase> <!-- Mobile_No_contain_negative_numbers -->
</testsuite> <!-- Master_Employee -->
